### PR TITLE
Selects indicator modules depending on whether or not it has made the target.

### PR DESCRIPTION
**Pull request by NickTUD**
**Important: only done now for AZC and Green module**

User story: As a Tygron user, I only want to improve my indicators for which I have not reached the target yet so I work as efficiently as possible.
- Implemented knowledge rules. Please check the dummy.pl file itself to see which have been implemented.
- Changed rules in the event module so the AZC and Green module only are entered if we have not reached their targets yet.

### CODE REVIEW

**Comment by dionytadema** [comment]
Looks good, however the azc module should be changed as it performes a similar check itself

**Comment by dionytadema** [comment]
has conflicts

**Comment by dionytadema** [comment]
i have changed the azc module, could you please fix conflicts

**Comment by dionytadema** [comment]
i fixed the conflicts

File: Municipality/dummy.pl [hunk 1]
```diff
@@ -18,8 +18,37 @@
 	actionlog/4,
 	my_stakeholder_id/1.
 
+%getMyID/1 - myID(<StakeholderID)
+%<StakeholderID> - Numeral ID for the stakeholder the agent is representing. Output variable.
+getMyID(StakeholderID):- stakeholder(StakeholderID,'Gemeente',_,_).
```
Comment (levilime) [inline Municipality/dummy.pl]: We have my_stakeholder_id percept so this won't be necessary

**Comment by levilime** [comment]
I like these changes and I think only going in a module if necessary is a nice touch. I think it's best if this refactor goes after #67 , #65 and #68 , because those are features

**Comment by dionytadema** [comment]
fixed conflict again, will merge as soon as travis completes as me and levi aprove

**Comment by levilime** [comment]
No I first will review the new changes before the merge, also this is Nick's ticket

**Comment by levilime** [comment]
As Nick is working on it currently

**Comment by dionytadema** [comment]
@levilime nick has not worked on it and as far as i know this in functional

**Comment by NickTUD** [comment]
Yeah there is no need for change on it now, If people would want to review this, that would be great.

**Comment by levilime** [comment]
@NickTUD I see, thanks Diony for fixing conflicts, I think the refactor is meaningful, but it also needs to be used in the code in this ticket. Please let these style changes propagate in the running code. For example RandomIndicator is not used yet as far as I have observed.

**Comment by levilime** [comment]
There are still functions in the dummy.pl that are not used in the code. I think this PR should also be about implementing them in the code.

**Comment by NickTUD** [comment]
@levilime I don't understand what you mean with this? Either the functions I implemented are used in an other function or they're used in the code.

**Comment by levilime** [comment]
Ah I see, ok I agree with the merge

**Comment by hmyeh** [comment]
Looks good :+1:

**Pull request merged**